mix.exs

```diff
@@ -27,7 +28,7 @@ defmodule Rig.Umbrella.Mixfile do
 
   defp versions do
     {map, []} = Code.eval_file("version", ".")
-    map
+    map 
```

Comment:
added whitespace.. If you're not doing this already, consider using format-on-save (i.e. `mix format`) with your editor - it fixes changes like this automatically